plugins/rtmp-services/data/locale/en-US.ini

```
@@ -8,3 +8,5 @@ UseAuth="Use authentication"
 Username="Username"
 Password="Password"
 ShowAll="Show all services"
+MultitrackVideo.Disclaimer="%1 automatically optimizes your settings to encode and send multiple video qualities to %2, powering the best possible viewer experience. Selecting this option will send %2 information about your computer and software setup."
```

re https://github.com/obsproject/obs-studio/pull/10634#pullrequestreview-2039534116
@Warchamp7, @RytoEX: so should we change the string to this
```suggestion
MultitrackVideo.Disclaimer="%1 automatically optimizes your settings to encode and send multiple video qualities to %2. Selecting this option will send %2 information about your computer and software setup."
```
?

---

Yes.